benchmarks/infra/stage-2/sample-tfvars/gpu-sample.tfvars

```diff
@@ -6,12 +6,12 @@ credentials_config = {
 
 # can be obtained from stage-1 by running:
 # terraform output -json  | jq '."project_id".value'
-project_id = "change-me"
+project_id = "$PROJECT_ID"
 
-bucket_name     = "${PROJECT_ID}-ai-gke-benchmark-fuse"
+bucket_name     = "ai-gke-benchmark-fuse"
 bucket_location = "US"
 
-output_bucket_name     = "${PROJECT_ID}-benchmark-output"
+output_bucket_name     = "benchmark-output"
```

Comment:
I think this has potential naming collisions? I thought GCS buckets have to be globally unique

Reply:
I think you're right, I gotta fix this before submitting - documentation here https://cloud.google.com/storage/docs/buckets#naming

My main goal for this change was it was easier to make 1 sample change vs many. I had a few steps where I missed a var to update :)

Wondering if we should prepend the project id in the module itself. Or possibly stack the variables that require a changed var next to each other at the top of the tfvars so it's more visible? WDYT?

Reply:
I'm down to prepend on their behalf, we could just leave a comment on the section of vars we will do that for, so they are aware.

Reply:
Sweet, I'm assuming it's something the jetstream example vars will also benefit from automatically, so I'll also update those configs when I get the prepending updated. Will ping when ready for review on that change :) thanks for catching this!

Reply:
updated commit to have prepended project id. Plan is to  do a follow up PR when we have time for automating the bucket naming schema (either in the modules or possibly in the orchestration end to end workflow)